conf/modules/px4io_flash.xml

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="px4io_flash">
+  <doc>
+    <description>Flashes the px4io f1 through the px4 bootloader.</description>
+  </doc>
+  <header>
+    <file name="px4io_flash.h"/>
+  </header>
+  <init fun="px4ioflash_init()"/>
+  <event fun="px4ioflash_event()"/>
+  <makefile target="ap">
+    <raw>
+        PX4IO_UART ?= UART6
+        PX4IO_BAUD ?= 1500000
+        PX4IO_UART_LOWER=$(shell echo $(PX4IO_UART) | tr A-Z a-z)
+
+        TELEM2_UART ?= UART3
+        TELEM2_BAUD ?= B115200
+        TELEM2_UART_LOWER=$(shell echo $(TELEM2_UART) | tr A-Z a-z)
+    </raw>
+    <file name="px4io_flash.c"/>
+    <define name="USE_$(PX4IO_UART)"/>
```

Comment:
Hmm, I got the PX4IO part to work but if I try to do the same for the TELEM2 port, it gives me 
modules/px4io_flash/px4io_flash.c: In function 'px4ioflash_event':
<command-line>:0:13: error: 'uart3' undeclared (first use in this function)